eng/pipelines/coreclr/templates/helix-queues-setup.yml

```diff
@@ -140,4 +140,8 @@ jobs:
     - ${{ if eq(parameters.platform, 'Browser_wasm') }}:
       - Ubuntu.1804.Amd64.Open
 
+    # Android
```

Comment:
It feels like the jobs should be in alphabetical order. If its possible, could we move this and WebAssembly (Maybe change to Browser wasm to follow the convention of os arch?) 
Also conventionally it looks like the jobs are split by arch, so split into Android x86 and Android x64?

Reply:
According to https://github.com/dotnet/runtime/issues/33381, only x64 is needed. I will remove x86.

Reply:
Fixed order.